src/graphology/src/utils.js

```diff
@@ -155,3 +155,101 @@ export function incrementalIdStartingFromRandomByte() {
     return i++;
   };
 }
+
+/**
+ * Takes the first `n` items from the given iterable.
+ * @param {Iterable} iterable
+ * @param {number} n
+ * @returns {Array}
+ */
+export function take(iterable, n) {
```

Comment:
At that point I feel like `Array.from` is sufficiently ubiquitous that we can drop the `take` util altogether, no? It is used twice in the codebase to convert a map keys iterator when the `Array.from` function is unavailable. What do you think?